PowerPointLabs/PowerPointLabs/FunctionalTestInterface.Impl/Controller/ColorsLabController.cs

```diff
@@ -320,6 +320,17 @@ public void ClearFavoriteColors()
             }
         }
 
+        public void ClearRecentColors()
+        {
+            if (_pane != null)
+            {
+                UIThreadExecutor.Execute((Action)(() =>

```

Comment:
Might it be better if this returns if (_pane == null), and UIThreadExecutor call is done on normal indent?

Reply:
Hmm... It might be better, however I actually followed the code for existing controllers for this part. There are several other labs whose FT controllers also do it this way. Maybe if we want to change it might be a better idea to change it for all the labs at once in a separate issue as code quality?

Reply:
Yeah, will need a separate issue.